aries-site/src/components/content/CardGrid.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Grid } from 'grommet';
+
+export const CardGrid = ({ children, ...rest }) => {
+  return (
+    <Grid columns="medium" gap="medium" {...rest}>
```

Comment:
Any reason we wouldn't want to use `aries-core Tiles` here?

Reply:
We can implement it that way, but if you look at `aries-core Tiles` its not doing a whole lot and could potentially be pruned as a component. I could be swayed if there is value that it provides that I am blind to at the moment.

Reply:
That mindset makes sense. I am fine leaving as is for now.